### PR TITLE
Add Czech to auto-py-to-exe

### PR DESCRIPTION
**Pull request by Matto58**
**Describe your change**

*Added Czech*

**New feature checklist**

- [x] Have you run the application to make sure your code runs
- [x] Is this not just a simple formatting change

**Translation checklist**

- [x] Have you run the application to make sure your code runs
- [x] Have you used the correct keys for your translations added (e.g. `en`) 
- [x] Have you made sure to leave a `,` at the end of each translation added
- [x] Have you added the language to `supportedLanguages` in alphabetical order
- [x] Have you added the language to the table in README.md in alphabetical order
- [ ] If you have translated the README, have you linked it in README.md

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -507,6 +542,7 @@ const translationMap = {
                 fr: 'Injectez du texte brut dans la commande générée.',
                 it: 'Inserisci testo forzatamente alla fine del comando generato.',
                 id: 'Masukkan teks mentah ke dalam command yang dibuat',
+                cz: 'Vložit text do vygenerovaného příkazu.'
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:545]: Can we get a comma at the end of this line please

File: auto_py_to_exe/web/js/i18n.js [hunk 2]
```diff
@@ -614,6 +654,10 @@ const translationMap = {
                     '    artikel ini untuk menyelesaikan masalah-masalah umum\n' +
                     '</a>\n' +
                     'yang mungkin.',
+                cz: 'Něco špatně s exe souborem? Přečtěte si\n' +
+                    '<a href="https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/?utm_source=auto_py_to_exe&utm_medium=application_link&utm_campaign=auto_py_to_exe_help&utm_content=bottom" target="_blank">\n' +
+                    '    tento příspěvek na jak opravit běžné chyby\n' +
+                    '</a>.\n' +
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:660]: Looks like you might be missing something after this

File: auto_py_to_exe/web/js/i18n.js [hunk 3]
```diff
@@ -717,6 +767,7 @@ const translationMap = {
                 fr: 'Options spécifiques pour Mac Os X',
                 it: 'Opzioni specifiche Mac Os X',
                 id: 'Pilihan Mac Os X',
+                cz: 'Mac OS X specifické nastavení',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:770]: There is an extra space after this comma

File: auto_py_to_exe/web/js/i18n.js [hunk 4]
```diff
@@ -862,6 +922,7 @@ const translationMap = {
                 fr: 'Toutes les valeurs précédemment insérées seront effacées.',
                 it: 'Tutti i valori inseriti sarranno cancellati.',
                 id: 'Semua nilai yang sebelumnya dimasukkan akan dihapus.',
+                cz: 'Všechny předchozí hodnoty budou smazány.'
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:925]: Can we get a comma at the end of this line please

File: auto_py_to_exe/web/js/i18n.js [hunk 5]
```diff
@@ -1067,6 +1132,10 @@ const supportedLanguages = [
         name: 'Turkish (Türkçe)',
         code: 'tr',
     },
+    {
+        name: 'Czech (Čeština)',
+        code: 'cz',
+    },
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:1138]: This should go under "Chinese Traditional"

File: auto_py_to_exe/web/js/i18n.js (hunk 1 above)
Comment (Matto58) [reply auto_py_to_exe/web/js/i18n.js]: Fixed!

**Review by brentvollebregt — APPROVED** [review]

**Comment by brentvollebregt** [comment]
Looks good, I'll get a release out soon

**Pull request merged**